scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -761,16 +761,10 @@
                       ;;     reflect *Cook*'s understanding of the state of the world at this point.
                       ;;     When this happens, users should never exceed their quota
                       user->usage-future (future (generate-user-usage-map (d/db conn)))
-                      offers (async/alt!!
-                               offers-chan ([offers]
-                                            (counters/dec! offer-chan-depth)
-                                            offers)
-                               trigger-chan ([_] [])
-                               :priority true)
                       ;; Try to clear the channel
                       offers (->> (repeatedly chan-length #(async/poll! offers-chan))
-                                  (filter nil?)
-                                  (reduce into offers))
+                                  (remove nil?)
```

Comment:
Do we need to reduce the counter?
[`(counters/dec! offer-chan-depth (count offers))`]